### PR TITLE
Pass FSWatcher.path to notification callback

### PR DESCRIPTION
**Pull request by aaron-em**
This pull request contains the change described in [https://github.com/3rd-Eden/fs.notify/issues/3](bigmonkeyboy's issue), which I'd argue is vital to the "not sucking hairy monkey balls" goal of the project, in that a callback, unable to tell on precisely which file it's receiving a notification, lacks enough utility that it can still be described as sucking at least one hairy monkey ball.

I acknowledge that there's no test included here, such as you requested from bigmonkeyboy. I'm not sure what you have in mind for such a test to cover, since your repo doesn't contain any preexisting tests on which to build. If you're unwilling to accept the pull request in its absence, then let me know what cases the test should include, and I'll submit another pull request with a test which meets those requirements.

### CODE REVIEW

**Comment by bigmonkeyboy** [comment]
Thanks - +1 for this. Happy to close my issue if this can be accepted.

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ var files = [
 ];
 
 var notifications = new Notify(files);
-notifications.on('change', function (file) {
-  console.log('file changed');
+notifications.on('change', function (file, event, path) {
```
Comment (aaron-em) [inline README.md:26]: I shouldn't have left this change in the pull request, but I'm willing to argue it's beneficial enough to be worth keeping around, in that it makes console logging considerably more useful.

**Comment by bigmonkeyboy** [comment]
no - libraries shouldn't console.log if possible - but should provide the information to do so... so thanks for tidying up.

**Comment by 3rd-Eden** [comment]
Thanks for pull request, 0.0.4 will be out in a minute

**Comment by aaron-em** [comment]
Fantastic, thanks!

**Pull request merged**